apps/website/src/config/env.server.mjs

```diff
@@ -3,6 +3,7 @@ import { z } from 'zod'
 export const envSchema = z.object({
```

Comment:
I would add check so it's not possible to import on client.

Reply:
done at https://github.com/status-im/status-web/pull/417/commits/419d7306b2fbc17e89118604fb0962f07233f742, thx